src/read.c

```diff
@@ -2729,35 +2730,23 @@ void ReadEvalError(void)
 struct SavedReaderState {
   UInt                userHasQuit;
   syJmp_buf           readJmpError;
-  UInt                intrCoding;
-  UInt                intrIgnoring;
-  UInt                intrReturning;
```

Comment:
We don't need to save/restore these here anymore, as the interpreter state is on the stack -- yay!